### PR TITLE
Improve coverage

### PR DESCRIPTION
**Pull request by vdiez**

### CODE REVIEW

File: test/unit/config.test.js [hunk 1]
```diff
@@ -391,6 +389,66 @@ describe('config', function () {
...
+      assert.equal(
+        getExecutableParams({ version: '4.7.0.2747' }).downloadUrl,
+        new URL(
+          'sonar-scanner-cli-' + '4.7.0.2747' + '-' + findTargetOS() + '.zip',
+          SONAR_SCANNER_MIRROR,
+        ),
+      );
+    });
+
+    it('should sanitize scanner version', function () {
```
Comment (ilia-kebets-sonarsource) [inline test/unit/config.test.js:441]:
```suggestion
    it('should sanitize the scanner version', function () {
```

File: test/unit/config.test.js [hunk 2]
```diff
@@ -391,6 +389,66 @@ describe('config', function () {
       );
     });
 
+    it('should not set http proxy if url is invalid', function () {
```
Comment (ilia-kebets-sonarsource) [inline test/unit/config.test.js:392]:
```suggestion
    it('should not set the http proxy if url is invalid', function () {
```

**Review by ilia-kebets-sonarsource — APPROVED** [review]
a few more suggestions otherwise LGTM

File: test/unit/config.test.js [hunk 3]
```diff
@@ -391,6 +389,66 @@ describe('config', function () {
...
+      assert.equal(
+        getExecutableParams({ version: '4.7.0.2747' }).downloadUrl,
+        new URL(
+          'sonar-scanner-cli-' + '4.7.0.2747' + '-' + findTargetOS() + '.zip',
+          SONAR_SCANNER_MIRROR,
+        ),
+      );
+    });
+
+    it('should sanitize the scanner version', function () {
```
[the remark this answers is not in the document]
Comment (vdiez) [reply test/unit/config.test.js:441]: yes, if it does not pass the validation regex we go back to default version

**Pull request merged**